roles/postgresql/defaults/main.yml

```diff
@@ -75,6 +75,11 @@ postgresql_jupyterhub_user: jupyterhub
 postgresql_jupyterhub_password: jupyterhub
 postgresql_jupyterhub_database: jupyterhub
 
+no_custom_schemas_users: >-
```

Comment:
I do not understand why you create this variable, you add `schema_generated` to `postgresql_generated_schemas` so why not using it ?